meeting-notes/2023-04-20-limitations.md

```diff
@@ -0,0 +1,29 @@
+# Sync meeting
+## Information
+**Date:** 2023-03-15  
+**Purpose:** Address some limitations 
+
+## Attendees
+People who attended:
+- [aquint-zama](https://github.com/aquint-zama)
+- [haroune-mohammedi](https://github.com/haroune-mohammedi)
+
+## Outcome
+We encountered the following limitations during https://github.com/open-dust/TFHE-UBIRIS/pull/2
+- Performance issues
+- We can't compare the iris sent from the client with a previously encrypted database of irises because of a limitation in Concrete-Python, to better understand this see the snippet below
+```python
+# Those values can't be encrypted
+data = [1, 2, 3]
+data_length = 3
+
+def f(x):
+    for i in range(data_length):
+        x = x + data[i]
+    return x
+```
+- We can't compare the resulting score to a threshold to decide whether the iris is matched or not, yet alone returing the encrypted ID of the matching iris
+
+To address theses, we agreed with @aquint-zama to keep the database of irises clear in the server for the moment and to return the score from the server and let another part of the system decide whether it's a match or miss.
```

Comment:
Secound thought, I think it's more representative to have both challenge and DB encrypted, that could be solved by sending the db with the challenge as inputs arguments.
Ok for the score